client/src/components/common/toggle-checkbox/toggle-checkbox.jsx

```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types';
+
+import styles from './styles.module.scss';
+
+const ToggleCheckbox = ({ checked, label, onChange }) => (
```

Comment:
```suggestion
const Checkbox = ({ checked, label, onChange }) => (
```
don't forget about name